### PR TITLE
Test/mobile crash

### LINKED ISSUE #625: Expand Smoke Test Crasher
Crash Test runs on Windows. It would be ideal to also cover Android and iOS (and from now on add to any new platform with native support).

### PR DESCRIPTION
**Pull request by vaind**
closes #625 

TODOs:
* [x] add to iOS tests
* [x] add to Android tests
* [x] fix android manifest (connecting using HTTP): https://stackoverflow.com/questions/45940861/android-8-cleartext-http-traffic-not-permitted
* ~rework CI so that both iOS & Android use the same process to create a project as windows does (i.e. create an empty project, build, add unity, build again, only test afterwards)~
* [x] or do a workaround - just update the sentry URL in the assets.

#skip-changelog

### CODE REVIEW

File: samples/unity-of-bugs/Assets/Scripts/SmokeTester.cs [hunk 1]
```diff
@@ -48,21 +48,7 @@ public void Start()
...
-            {
-                arg = "smoke";
-                Debug.Log("Key " + key + " found on Info.plist starting Smoke test.");
-            }
-            else
-            {
-                Debug.Log("To run Smoke Test, please add key " + key + " into Info.plist");
-            }
-        }
+        arg = getTestArgObjectiveC();
```
Comment (bruno-garcia) [inline samples/unity-of-bugs/Assets/Scripts/SmokeTester.cs:51]: Does `Environment.GetCommandLineArgs();` return already the same thing? Unity might have done data binding here

File: samples/unity-of-bugs/Assets/Scripts/SmokeTester.cs [hunk 2]
```diff
@@ -93,6 +79,11 @@ public void Start()
         }
     }
 
+#if UNITY_IOS
```
Comment (bruno-garcia) [inline samples/unity-of-bugs/Assets/Scripts/SmokeTester.cs:82]: Best to throw below the non P/Invoke methods (so we can read through `InitSentry` and others before finding the helper methods

Comment (bruno-garcia) [inline samples/unity-of-bugs/Assets/Scripts/SmokeTester.cs:82]: Best to throw below the non P/Invoke methods (so we can read through `InitSentry` and others before finding the helper methods

File: samples/unity-of-bugs/Assets/Scripts/SmokeTester.cs (hunk 1 above)
Comment (vaind) [reply samples/unity-of-bugs/Assets/Scripts/SmokeTester.cs:51]: I've tried and it doesn't work. I'll add a comment

[the remark this answers is not in the document]
Comment (vaind) [reply samples/unity-of-bugs/Assets/Scripts/SmokeTester.cs:52]: https://github.com/getsentry/sentry-unity/pull/630#discussion_r824287886

File: samples/unity-of-bugs/Assets/Plugins/Sentry/SentryCliOptions.asset [hunk 3]
```diff
@@ -0,0 +1,19 @@
+%YAML 1.1
```
Comment (bitsandfoxes) [inline samples/unity-of-bugs/Assets/Plugins/Sentry/SentryCliOptions.asset:1]: It looks like I forgot to add SentryCliOptions to .gitignore

File: samples/unity-of-bugs/Assets/Scripts/SmokeTester.cs (hunk 1 above)
[the remark this answers is not in the document]
Comment (bruno-garcia) [reply samples/unity-of-bugs/Assets/Scripts/SmokeTester.cs:52]: not sure why I sent this twice, sorry. Thanks for trying

**Review by bruno-garcia — APPROVED** [review]

File: samples/unity-of-bugs/Assets/Plugins/Sentry/SentryCliOptions.asset (hunk 3 above)
Comment (vaind) [reply samples/unity-of-bugs/Assets/Plugins/Sentry/SentryCliOptions.asset]: removed & added to gitignore

**Pull request merged**